app/assets/v2/css/forms/button.css

```diff
@@ -17,7 +17,9 @@
 }
 
 .button:disabled,
```

Comment:
This styles will overwrite all disabled buttons with that class .button. Also the css patterns here is BEM since we are using bootstrap I don't recommend mixing styles/names. Maybe extending the current btn will be a better practice.

Reply:
> This styles will overwrite all disabled buttons with that class .button. 

this is by design.  the current styles dont visibly show that the disabled buttons are disabled

Reply:
i'm not sure i understand what youre proposing

Reply:
Is not a big deal since is mixed in other areas of the app but that styles are in a BEM way to write css since bootstrap is not BEM maybe a better thing is to create a .btn variation (extend bootstrap) and not jumping to a different way to create class, also .btn:disabled already is styled.
And since I saw our app have some mixed class with BEM and also very similiar to others in bootstrap way.
I think that one can be styled ```.btn .btn-gc-purple``` will have a similar effect than ```.button .button--primary```
Also another point is ```:disabled``` is not an ```<a>``` valid attr. That is just for button element and using bootstrap .btn will style :disabled attr and .btn.disabled to be used in links elements.

Reply:
![image](https://user-images.githubusercontent.com/1358093/43227487-39ca8026-9035-11e8-81ef-9269417ba5ac.png)